app/assets/javascripts/discourse/app/components/user-menu/menu.js

```diff
@@ -175,6 +221,10 @@ export default class UserMenu extends Component {
 
   constructor() {
     super(...arguments);
+    this.site.set(
+      "unassignedNotificationTypes",
+      this._unassignedNotificationTypes
+    );
```

Comment:
(note this should no longer be needed if we make the changes I described above about passing `notificationTypes` to the panel component)